File: app/src/androidTest/java/com/test/news/cucumberTest/espresso/login/LoginScreenRobot.java
package com.test.news.cucumberTest.espresso.login;

import androidx.test.rule.ActivityTestRule;

import com.test.news.features.login.presentation.LoginActivity;

public class LoginScreenRobot {

    public void launchLoginScreen(ActivityTestRule<LoginActivity> testRule) {
        testRule.launchActivity(null);
    }

}

File: app/src/androidTest/java/com/test/news/cucumberTest/espresso/login/LoginScreenRobot.kt
package com.test.news.cucumberTest.espresso.login

import androidx.test.rule.ActivityTestRule
import com.test.news.features.login.presentation.LoginActivity

class LoginScreenRobot {

    fun launchLoginScreen(testRule: ActivityTestRule<LoginActivity>) {
        testRule.launchActivity(null)
    }

}
